var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -85,7 +92,7 @@ def cmake_args(self):
                 '-DUSE_XSDK_DEFAULTS:BOOL=ON',
                 '-DXSDK_ENABLE_PYTHON:BOOL={0}'.format(
                     'ON' if '+python' in spec else 'OFF'),
-                '-DTasmanian_ENABLE_MPI:BOOL={0}'.format(
+                '-DTPL_ENABLE_MPI:BOOL={0}'.format(
```

Comment:
Does this CMake value exist for all versions of Tasmanian?

Reply:
It exists in 6.0 (the spack package was incorrect), `+mpi` conflicts with 5.1 and older.